mcsema/Arch/ABI.cpp

```diff
@@ -455,14 +493,73 @@ static llvm::Function *WriteIntToMemFunc(uint64_t size_bytes) {
   }
 }
 
-llvm::Value *CallingConvention::LoadNextArgument(llvm::BasicBlock *block,
-                                                 llvm::Type *goal_type) {
+llvm::Value* InsertIntoVector(llvm::BasicBlock *block,
+                              llvm::Value *base_value,
+                              llvm::Value *reg_ptr,
+                              size_t count, size_t start=0) {
+
+  llvm::IRBuilder<> ir(block);
+  for (size_t i = 0; i < count; ++i) {
+    auto offset = ir.CreateGEP(reg_ptr, GetConstantInt(64, i));
+    auto load = ir.CreateLoad(offset);
+    base_value = ir.CreateInsertElement(base_value, load,
+        GetConstantInt(64, i + start));
```

Comment:
Either move all args to next line, or align `GetConst..` with `base_value`.

The line splitting of arguments is either everything other lines, with 4-spaces indent, or some stuff on the same line and some stuff on other lines, with the first arg on each new line aligned with the first arg to the function.